format/index.md

```diff
@@ -1453,6 +1481,14 @@ constraints (such as a uniqueness constraint on a property other than `id`).
 A server **MUST** return `409 Conflict` when processing a `PATCH` request in
 which the resource object's `type` and `id` do not match the server's endpoint.
 
+A server **SHOULD** return `409 Conflict` when processing a `PATCH` request in
+which the resource object's `revision` does not match the server's endpoint
+and the server was *unable to resolve* the `revision` mismatch for updating
```

Comment:
I would remove this and the following lines, see https://github.com/json-api/json-api/pull/824#issuecomment-140977824